### PR TITLE
fix(intezer): set URL dbot score

### PR DESCRIPTION
**Pull request by davidt99**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fix: Set dbot score according to the URL analysis verdict
Feature: Add get IOCs command

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 1]
```diff
@@ -401,6 +399,45 @@ def get_analysis_metadata_command(intezer_api: IntezerApi, args: dict) -> Comman
...
+
+    iocs = analysis.iocs
+    readable_output = ''
+    if iocs:
+        network_iocs = iocs.get('network')
+        if network_iocs:
+            readable_output += tableToMarkdown('Network IOCs', network_iocs)
+        files_iocs = iocs.get('files')
+        if files_iocs:
+            readable_output += tableToMarkdown('Files IOCs', files_iocs)
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:423]: Would be better to use Walrus operator:
```suggestion
    if iocs:
        if network_iocs := iocs.get('network'):
            readable_output += tableToMarkdown('Network IOCs', network_iocs)
        if files_iocs := iocs.get('files'):
            readable_output += tableToMarkdown('Files IOCs', files_iocs)
```

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 2]
```diff
@@ -307,6 +302,7 @@ def get_analysis_code_reuse_command(intezer_api: IntezerApi, args: dict) -> Comm
                                                 composed_analysis_id=analysis_id,
                                                 sha256='',
                                                 source='',
+                                                extraction_info=None,
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:305]: Can you please explain why did you add the extraction_info arg here?

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml [hunk 3]
```diff
@@ -291,7 +291,25 @@ script:
...
+      deprecated: false
+      description: Get analysis IOCs.
+      execution: false
+      outputs:
+        - contextPath: Intezer.Analysis.ID
+          description: The composed analysis ID
+          type: string
+        - contextPath: Intezer.Analysis.IOCs
+          description: The Analysis IOCs
+          type: Unknown
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml:311]: Is the type really unknown?

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py [hunk 4]
```diff
@@ -670,6 +673,137 @@ def test_get_analysis_metadata_command_analysis_doesnt_exist(requests_mock):
...
+    )
+
+    args = dict(analysis_id=analysis_id)
+
+    # Act
+    command_results = get_analysis_iocs_command(intezer_api, args)
+
+    # Assert
+    outputs = command_results.outputs['Intezer.Analysis(obj.ID == val.ID)']
+    assert outputs['ID'] == analysis_id
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py:751]:
```suggestion
    assert outputs.get('ID') == analysis_id
```

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py [hunk 5]
```diff
@@ -670,6 +673,137 @@ def test_get_analysis_metadata_command_analysis_doesnt_exist(requests_mock):
...
+    assert command_results.readable_output == 'No IOCs found'
+    assert outputs['IOCs'] is None
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py:786]:
```suggestion
    assert outputs.get('ID') == analysis_id
    assert command_results.readable_output == 'No IOCs found'
    assert outputs.get('IOCs') is None
```

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py [hunk 6]
```diff
@@ -229,7 +231,8 @@ def test_check_analysis_status_and_get_results_url_command_single_success(reques
                 'analysis_id': analysis_id,
                 'summary': {
                     'title': 'malicious',
-                    'verdict_name': 'malicious'
+                    'verdict_name': 'malicious',
+                    'verdict_type': 'malicious'
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py:235]: I think it would be great to add to this test (or to another test) verification of the dbot score that was outputted to the context. I mean, I don't think that this test checks the dbot score that was generated (it checks only the dbot indicator).

File: Packs/Intezer/Integrations/IntezerV2/README.md [hunk 7]
```diff
@@ -688,3 +698,94 @@ size_in_bytes	        245760
 ssdeep	                ssdeep
 ```
 
+### intezer-get-analysis-iocs
+
+***
+Gets the list of network and files IOCs of a specific analysis id.
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/README.md:704]: I think that this is a better description than the one you added to the YML - worth adding that one to the YML file instead.

File: Packs/Intezer/Integrations/IntezerV2/README.md [hunk 8]
```diff
@@ -688,3 +698,94 @@ size_in_bytes	        245760
...
+#### Context Output
+
+| **Path**                              | **Type**  | **Description**          |
+|---------------------------------------|-----------|--------------------------|
+| Intezer.Analysis.ID                   | string    | The composed analysis ID | 
+| Intezer.Analysis.IOCs                 | Unknown   | The Analysis IOCs        | 
+
+#### Context Example
+
+```json
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/README.md:725]: Please verify that all the IPs and file hashes here are not secrets - would be better to use something like 1.1.1.1. 

In addition, I think the example could be shorter - there is no need to add to json more than one file example and one network example.

File: Packs/Intezer/ReleaseNotes/1_3_3.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+#### Integrations
+##### Intezer v2
+- Bug Fixes
+  - Fixed an issue in URL score setting.
+- New Commands
+  - `intezer-get-analysis-iocs` - Get analysis IOCs command.
```
Comment (ShacharKidor) [inline Packs/Intezer/ReleaseNotes/1_3_3.md:6]:
```suggestion
#### Integrations
##### Intezer v2
- Fixed an issue where the dbot score of URLs was set incorrectly.
- Added the ***intezer-get-analysis-iocs*** command which gets the list of network and files IOCs of a specific analysis id.
```

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @davidt99 - Good work :) 
Thanks for contributing!

I added a few comments - please see them below. 

Let's schedule a short demo :)
I'm available on slack for every question.

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py (hunk 2 above)
Comment (davidt99) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:305]: This is due to a change in our python SDK that was auto-updated

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 10]
```diff
@@ -373,7 +371,7 @@ def get_analysis_metadata_command(intezer_api: IntezerApi, args: dict) -> Comman
             return _get_missing_analysis_result(analysis_id=str(analysis_id))
         elif error.response.status_code == HTTPStatus.CONFLICT:
             return _get_analysis_running_result(analysis_id=str(analysis_id))
-
+        raise
```
[the remark this answers is not in the document]
Comment (davidt99) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:374]: The same reason as my previous point

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py (hunk 6 above)
Comment (davidt99) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2_test.py:235]: Done

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml (hunk 3 above)
Comment (ShacharKidor) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml:311]: So would be better to write 'Dict' instead. 
(change it in the readme file as well).

File: Packs/Intezer/Integrations/IntezerV2/README.md (hunk 8 above)
Comment (davidt99) [reply Packs/Intezer/Integrations/IntezerV2/README.md:725]: I removed IP but kept network IOCs of two types: IP and domain
